README.md

```diff
@@ -38,12 +42,18 @@ Python](https://www.python.org/dev/peps/pep-0020/).
 
 ## 1. Formality-Core
 
-Formality-Core is the minimal core behind Formality. If you see Formality as a
-programming language, then Core is the minimal amount of features required to
-derive everything else as libraries. If you see it as a proof language, then
-Core is the set of axioms from which all of mathematics derive. On this section,
-we'll specify it informally. Reference implementations using popular languages
-are provided later. 
+Formality-Core is the minimal semantic core behind Formality. If you look at
+Formality as a programming language, then Core is the minimal amount of features
+required to derive everything else as libraries. If you look at it as proof
+language, then Core is the set of axioms underlying the proof system. The
+[Curry-Howard
+correspondence](https://en.wikipedia.org/wiki/Curry%E2%80%93Howard_correspondence)
```

Comment:
No links! No citing Curry-Howard too. We could mention somewhere the correspondence between programs and proofs, but I'd like to do that later. I want to postpone as much as possible the point where people feel overwhelmed by jargon, so, ideally, the first sections should act like a welcoming corridor. A common developer should be able to read as much as possible without hitting those words. So let's avoid that at this point of the document.